src/pseudopeople/constants/noise_type_metadata.py

```diff
@@ -27,3 +27,14 @@
 
 
 INT_COLUMNS = ["age", "wages", "mailing_address_po_box"]
+
+
+HOUSING_TYPE_GUARDIAN_DUPLICATION_RELATONSHIP_MAP = {
+    "Carceral": "Institutionalized group quarters population",
+    "Nursing home": "Institutionalized group quarters population",
+    "Other institutional": "Institutionalized group quarters population",
+    "College": "Noninstitutionalized group quarters population",
+    "Military": "Noninstitutionalized group quarters population",
+    "Other noninstitutional": "Noninstitutionalized group quarters population",
+    "Household": "Other relative",
```

Comment:
Is other relative correct? If their guardian was the household-head, shouldn't it be some sort of child relationship (i.e. biological child, step-child, etc.), and if not, shouldn't we be able to deduce the relationship in many cases?